tox.ini

```diff
@@ -27,7 +27,7 @@ commands =
 deps =
   coverage
   django111: Django>=1.11,<2.0
-  django20: Django>=2.0,<2.1
+  django20: Django>=2.0,<2.2
```

Comment:
Ah yes okay - the lines don't give much guidance... but you would have to ADD django 2.1 testing in the matrix, not overwrite Django 2.0 testing. Let me get that fixed...